pkgs/applications/virtualization/xen/packages.nix

```
@@ -24,10 +24,15 @@ let
       '';
     };
   };
+  # Broken with python 3.12+ when using internal QEMU due to https://github.com/NixOS/nixpkgs/issues/253751
+  python3 = python311;
```

```suggestion
  standard = {
    ...
    python3 = python311;
  };
  slim = {
    ...
    python3 = python;
  };
```
...
```nix
inherit (standard) meta python3;
...
inherit (slim) meta python3;
```
If we can build `xen-slim` with an unpinned python, we definitely should.

---

Done.